src/fhirtypes/common.ts

```diff
@@ -114,7 +114,63 @@ export function setImpliedPropertiesOnInstance(
             // Implied paths are found via the index free path
             finalPath.replace(/\[[-+]?\d+\]$/g, '')
           );
-          [finalPath, ...impliedPaths].forEach(ip => {
+          const allPaths = [finalPath, ...impliedPaths];
+          // Transform paths such that any required reslices may be satisfied by existing slices
+          const parents = associatedEl.getAllParents().slice(0, -1).reverse(); // [oldest ancestor, ... grandparent, parent]
+          const allRequiredSlices = parents.map(parent => {
+            const elementWithSlices = parent.slicedElement() ?? parent;
+            const slices = elementWithSlices.getSlices();
+            // parent's effective min (min for element with that path) = parent's - (all slices that start with "parent's name/")
+            const effectiveMins = slices.map(
+              slice =>
+                slice.min -
+                sumBy(
+                  slices.filter(s => s.sliceName.startsWith(`${slice.sliceName}/`)),
+                  'min'
+                )
+            );
+            const slicesWithMins = zipWith(slices, effectiveMins, (a, b) => ({
+              sliceName: a.sliceName,
+              min: b
+            }));
+            return slicesWithMins;
```

Comment:
Ohh yeah I think you're right. Although I want to double check with @mint-thompson before updating this!

Reply:
@jafeltra - did you get a chance to check w/ @mint-thompson on this?  I see that your commit did not make a change here, but I'm not sure if that's because you decided it is better to use `zipWith` after all or if you just didn't get a chance to check w/ Mint.

Reply:
I should have updated on here, but I thought I had at one point, but then everything got sidetracked while trying to rework the other branch. We're hoping to pull updates into this branch today, which was when I was going to see if `zipWith` is even still in the current implementation. So - hadn't really decided yet, but hopefully coming soon!

Reply:
@jafeltra - so is this PR still in progress then?  One of the several that are tied to the overall slicing work you're doing with Mint?

Reply:
@cmoesel - correct. This one isn't ready for re-review yet. I believe originally the issues that Mint had found on the `cimpl-865-soft-index-with-slices` branch/PR were also issues in this branch, and the fix affected the changes we had made here. So we're hoping to be able to pull the updates we have in the soft indexing with slices branch over to this branch as well.

Reply:
Yeah I'm good with that! I'll do that now.